### ISSUE TITLE
iframe requires title attribute

### ISSUE THREAD

**Issue by SavvasStephanides** [open-sdg/open-sdg#740]
**Please complete the following information**
Data repository URL: github.com/ONSdigital/sdg-data
Data or metadata issue: Metadata
Input format: YAML (Markdown)

**Describe the bug**
The `iframe` on the embed on [17.13.1](https://sustainabledevelopment-uk.github.io/17-13-1/) doesn't have a `title` tag. This is required for accessibility. [This file](https://github.com/ONSdigital/sdg-data/blob/develop/meta/17-13-1.md) needs to be updated

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I think this should be moved to the onsdigital/sdg-indicators repo right?

**Comment by SavvasStephanides** [comment]
@brockfanning @LucyGwilliamAdmin I wasn't sure since the "data repository URL" field lets you specify a full URL, including owner.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning @SavvasStephanides I think this is related to #383 which is about being able to add attributes to an iframe created using Pym parent.

However, I recently had a request from someone in UK data team for an iframe to be embedded and I was thinking maybe we could have embedded_feature_iframe instead of/aswell as embedded_feature_url - I did a quick test on a feature branch by overwriting some of the files - I will create a branch here in open-sdg though so can see diff

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning @SavvasStephanides have created PR but mainly for the purpose to show how this could work, as I know we haven't really discussed it.

Feature branch here:

http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/embed-test-v1/3-b-1/

**Comment by SavvasStephanides** [comment]
@LucyGwilliamAdmin I wasn't aware that you embed iframes using PymJS. In [this metadata file](https://github.com/ONSdigital/sdg-data/blob/develop/meta/17-13-1.md) it seems that you've hardcoded the embed, and it was just be a matter of updating the file to include a `title` tag. 

I saw your feature branch, but the `iframe` still doesn't have a `title` tag. Has this been implemented yet? (Not sure)

**Comment by LucyGwilliamAdmin** [comment]
@SavvasStephanides 17.13.1 is slightly different as the iframe is embedded as the main content so that is something that can be (and still needs to be) done in sdg-data.

The functionality I'm talking about is for the other scenario where iframes are embedded in new data tab, see the indicator on this feature branch as an example:
http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/embed-test-v1/3-b-1/

**Comment by SavvasStephanides** [comment]
@LucyGwilliamAdmin Looks good and just seen the raw `.md` file. Is the `embedded_feature_title` part going to be the title of the `iframe` somehow?

Not sure if I got my point across correctly, but by "title", I mean adding a `title` attribute to the `iframe` itself (not adding a heading at the top of the embed). Eg. 

```html
<iframe title="Childhood Vaccination Coverage Statistics"></iframe>
```

Otherwise Pa11y would keep complaining

**Comment by LucyGwilliamAdmin** [comment]
@SavvasStephanides `embedded_feature_title` is separate from the iframe. I understand what you mean, so far was just thinking about the functionality so that we are able to add a title attribute in the scenario where feature is embedded in a tab, as currently not possible.

**Comment by LucyGwilliamAdmin** [comment]
@SavvasStephanides @brockfanning I think this can be closed now that there's functionality to add title attribute in both scenarios

Savvas - I've added title attribute for 17.13.1 - about to push to live when possible, so hopefully shouldn't be getting that error on accessibility report now
